src/shared/components/Checkout/Checkout.styles.tsx

```diff
@@ -58,9 +80,23 @@ export const Step = styled.div`
   cursor: pointer;
   display: grid;
   grid-auto-flow: column;
-  grid-gap: 12px;
   justify-content: space-between;
   align-items: center;
+  padding: ${sizes(2)};
+  :last-of-type {
+    position: relative;
+    ::after {
```

Comment:
It seems like this horizontal line is vertically misaligned  a little bit. In design there is 16px of space above and below the line